tests/by-util/test_stat.rs

```diff
@@ -304,6 +304,19 @@ fn test_stdin_pipe_fifo2() {
         .succeeded();
 }
 
+#[test]
+#[cfg(all(unix, not(target_os = "android")))]
+fn test_stdin_with_fs_option() {
+    // $ stat -f -
+    // stat: using '-' to denote standard input does not work in file system mode
```

Comment:
Instead of using a comment you can express the same by codifying it with `stderr_contains`.

Reply:
Yep, added it now